### PR TITLE
rerun vndr

### PR DESCRIPTION
**Pull request by AkihiroSuda**
* run latest vndr so as to collect more LICENSE files
 * remove unused packages
 * vendor github.com/philhofer/fwd with LICENSE.md (MIT)
 * vendor github.com/bsphere/le_go with LICENSE (MIT)

Before this PR:
```console
$ make validate
WARNING: could not find copyright information for github.com/kr/pty
WARNING: could not find copyright information for github.com/mattn/go-shellwords
WARNING: could not find copyright information for github.com/docker/libkv
WARNING: could not find copyright information for github.com/opencontainers/go-digest
WARNING: could not find copyright information for github.com/philhofer/fwd
WARNING: could not find copyright information for github.com/aws/aws-sdk-go
WARNING: could not find copyright information for github.com/bsphere/le_go
WARNING: could not find copyright information for github.com/docker/containerd
WARNING: github.com/mreiferson/go-httpclient is vendored but unused
WARNING: bitbucket.org/ww/goautoneg is vendored but unused
WARNING: could not find copyright information for github.com/spf13/cobra
WARNING: could not find copyright information for github.com/docker/go-metrics
WARNING: could not find copyright information for github.com/xeipuuv/gojsonpointer
WARNING: could not find copyright information for github.com/xeipuuv/gojsonreference
WARNING: could not find copyright information for github.com/xeipuuv/gojsonschema
```

After:
```console
$ make validate
WARNING: could not find copyright information for github.com/mattn/go-shellwords
```

(this is not a legal issue, see `vendor/github.com/mattn/go-shellwords/README.md` (MIT) )

addendum to https://github.com/docker/docker/pull/31076

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM 🐯
/cc @thaJeztah @icecrime @LK4D4

**Comment by AkihiroSuda** [comment]
mm, need to install the latest vndr on Jenkins..
https://jenkins.dockerproject.org/job/Docker-PRs-vendor/2936/console

**Comment by vdemeester** [comment]
@AkihiroSuda  ah, it's in `binaries-commits` 😉

**Comment by AkihiroSuda** [comment]
thank you, updated PR

File: vendor/cloud.google.com/go/license_test.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (thaJeztah) [inline vendor/cloud.google.com/go/license_test.go]: Looks like it's vendoring a `_test` file here @LK4D4 ?

**Comment by thaJeztah** [comment]
Looks like the matching needs a small tweak in vndr (https://github.com/docker/docker/pull/31566#pullrequestreview-25225672)

**Comment by AkihiroSuda** [comment]
updated PR with LK4D4/vndr@19220953c4a45310a4c404b7905154e29120249e

**Review by vdemeester — APPROVED** [review]
still LGTM 🐸

**Comment by thaJeztah** [comment]
For reference; for the version bumps;

for https://github.com/bsphere/le_go/ (includes https://github.com/philhofer/fwd/commit/8fd9a4b75098b2125fe442e48a3ffbf738254e13)
https://github.com/philhofer/fwd/compare/899e4efba8eaa1fea74175308f3fae18ff3319fa...98c11a7a6ec829d672b03833c3d69a7fae1ca972

for https://github.com/bsphere/le_go/ (relates to https://github.com/docker/docker/issues/31020)
https://github.com/bsphere/le_go/compare/d3308aafe090956bc89a65f0769f58251a1b4f03...7a984a84b5492ae539b79b62fb4a10afc63c7bcf

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
ignoring power and z CI for now, looks like that's still being worked on

**Pull request merged**